packages/mobile/src/middlewares/http/http-error-handler.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpError } from '~/common/enums/enums';
+import { t } from 'i18next';
+import { notification } from '~/services/services';
+
+const httpErrorHandler = (response: Response): void => {
```

Comment:
This is not exactly what I meant actually. Please, take a look at redux [middlewares](https://redux.js.org/understanding/history-and-design/middleware)

What we need is to create a simple middleware which would look for `action.error` and notify in case it appears.

And also we don't have to check against all the existed HTTP error codes. We already receiving that error from backend. So we just have to display it without overriding.

Reply:
Refactored